process.py

```diff
@@ -7,4 +7,4 @@
 g.app.get_status_codes()
 g.app.get_data()
 g.app.write_json()
-g.app.upload_file_to_s3()
+# g.app.upload_file_to_s3()
```

Comment:
I guess we want to uncomment this chipolata?

Reply:
hahahahahah yep - thanks will